web/site/devops/vaults.env

```diff
@@ -7,4 +7,9 @@ AUTH_API_KEY="op://firebase/$APP_ENV/base/API_KEY"
 AUTH_DOMAIN="op://firebase/$APP_ENV/base/AUTH_DOMAIN"
 AUTH_PROJECT_ID="op://firebase/$APP_ENV/base/PROJECT_ID"
 AUTH_MESSAGING_SENDER_ID="op://firebase/$APP_ENV/business-ar/MESSAGING_SENDER_ID"
-AUTH_APP_ID="op://firebase/$APP_ENV/business-ar/APP_ID"
\ No newline at end of file
+AUTH_APP_ID="op://firebase/$APP_ENV/business-ar/APP_ID"
+
+#vaults keycloak
+NUXT_KEYCLOAK_AUTH_URL="op://keycloak/$APP_ENV/base/KEYCLOAK_AUTH_BASE_URL"
+NUXT_KEYCLOAK_REALM="op://keycloak/$APP_ENV/base/KEYCLOAK_REALMNAME"
+NUXT_KEYCLOAK_CLIENTID="op://keycloak/$APP_ENV/business-ar/UI_KEYCLOAK_RESOURCE_NAME"
```

Comment:
Is this the correct setting? I have keycloak running with my local env but in the /gcbrun its failing with "missing client id"